contracts/stake/src/wasm.rs

```diff
@@ -150,18 +158,26 @@ impl StakeContract {
 
         // verify signature
         let message = Self::withdraw_sign_message(counter, address, nonce);
-        let valid_signature = rusk_abi::verify_bls_sign(signature, pk, message);
+        let valid_signature =
+            rusk_abi::verify_bls(message, APK::from(&pk), signature);
 
         if !valid_signature {
             panic!("Invalid signature!");
         }
 
         // make call to transfer contract to mint the reward to the given
         // address
-        let transfer = rusk_abi::transfer_contract();
-        let transaction = Call::mint(address, reward, nonce).to_transaction();
-
-        dusk_abi::transact_raw(self, &transfer, &transaction, 0)
+        let transfer = rusk_abi::transfer_module();
+        let transaction = RawTransaction::new(
+            "withdraw",
+            Mint {
+                address,
+                value: reward,
+                nonce,
+            },
+        );
+
+        self.transact_raw(transfer, transaction)
```

Comment:
Same issue here. You'll want to call `mint` instead of `withdraw`, and probably use `transact` instead of `transact_raw`. Curiously, there's no new structure (`Mint2`) used here.